libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -303,11 +338,13 @@ export class PageComponent
 
   private removeWrapper() {
     const parent = this.elementRef.nativeElement.parentNode;
-    const header = this.elementRef.nativeElement.childNodes[0];
-    const content = this.elementRef.nativeElement.childNodes[1];
+    const header = this.ionHeaderElement.nativeElement;
+    const content = this.ionContentElement.nativeElement;
+    const footer = this.ionFooterElement.nativeElement;
     this.renderer.removeChild(parent, this.elementRef.nativeElement);
     this.renderer.appendChild(parent, header);
     this.renderer.appendChild(parent, content);
+    this.renderer.appendChild(parent, footer);
```

Comment:
Consider inlining consts as they are only used once:
```suggestion
    this.renderer.removeChild(parent, this.elementRef.nativeElement);
    this.renderer.appendChild(parent,  this.ionHeaderElement.nativeElement;);
    this.renderer.appendChild(parent, this.ionContentElement.nativeElement);
    this.renderer.appendChild(parent, this.ionFooterElement.nativeElement);
```